src/services/conversionRates/getConversionRatesService.js

```diff
@@ -203,18 +203,36 @@ const getHourlyRateCoingecko = async (
 
   return rate;
 };
+
+const findNewestData = tokenCompareHistoryResponse => {
+  return (
+    tokenCompareHistoryResponse &&
+    tokenCompareHistoryResponse.Data &&
+    // when there is no result and token is invalid, the resp.Data is {} not an array
+    Array.isArray(tokenCompareHistoryResponse.Data) &&
+    tokenCompareHistoryResponse.Data.length > 0 &&
+    tokenCompareHistoryResponse.Data.sort((a, b) => {
+      return b.time - a.time;
+    })[0]
+  );
+};
```

Comment:
If you call cryptocompare https://min-api.cryptocompare.com/data/histohour?fsym=DAI&tsym=BTC&toTs=1607748079&extraParams=giveth&limit=1

you will see the response is like this :

```
{
    "Response": "Success",
    "Type": 100,
    "Aggregated": false,
    "TimeTo": 1607745600,
    "TimeFrom": 1607742000,
    "FirstValueInArray": true,
    "ConversionType": {
        "type": "invert_divide",
        "conversionSymbol": "ETH"
    },
    "Data": [
        {
            "time": 1607742000,
            "high": 0.00005494,
            "low": 0.00005477,
            "open": 0.00005481,
            "volumefrom": 41444.98,
            "volumeto": 2.274,
            "close": 0.0000549,
            "conversionType": "invert",
            "conversionSymbol": ""
        },
        {
            "time": 1607745600,
            "high": 0.00005473,
            "low": 0.00005441,
            "open": 0.00005495,
            "volumefrom": 13936.37,
            "volumeto": 0.762,
            "close": 0.00005468,
            "conversionType": "invert_divide",
            "conversionSymbol": "ETH"
        }
    ],
    "RateLimit": {},
    "HasWarning": false
}
```


@aminlatifi  and none of objects doesn't have field `time` equals to timestamp that we send (in this example :1607748079),
so finding object with` time` equals to `timestampMS` always return undefined, and for solving this problem I sort the array and return the newest one, 
this function have tests , you can check it out